app/code/Magento/Catalog/Block/Adminhtml/Product/Edit/Tab/Inventory.php

```diff
@@ -168,7 +176,7 @@ public function getConfigFieldValue($field)
      */
     public function getDefaultConfigValue($field)
     {
-        return $this->stockConfiguration->getDefaultConfigValue($field);
+        return $this->json->unserialize($this->stockConfiguration->getDefaultConfigValue($field));
```

Comment:
The `getDefaultConfigValue` method is not expected to be used for serialized values only.
I guess we should avoid applying specific changes for such generic methods.